content/courses/onchain-development/intro-to-onchain.md

```diff
@@ -11,133 +11,192 @@ description:
+
+#### 4. Global Config Example
+
+- **Seeds**: `["config"]`
+- **Purpose**: Store global settings for your entire program.
+- **Benefits**: Centralize program-wide configurations in an easily accessible
+  location.
+
+#### Key Benefits of Using PDAs
+
+1. **Consistency**: The same seeds always produce the same address, ensuring
+   data can be reliably located.
+2. **Uniqueness**: PDAs are unique to your program, preventing conflicts with
+   other programs.
+3. **Flexibility**: You can create complex data structures by carefully choosing
+   your seeds.
+4. **Efficiency**: Quick and deterministic access to data based on known
+   parameters (seeds).
 
 ### Solana instructions need to specify all the accounts they will use
 
-As you may already know, Solana is fast because it can process transactions that
-don't overlap at the same time. I.e., just like in the real world, Alice sending
-to Bob doesn't stop Chris from sending something to Diana. Your front-end apps
-need to specify the addresses of all the accounts they will use.
+As you may already know, Solana is fast because it can process non-overlapping
+transactions simultaneously. For example, Alice sending tokens to Bob doesn’t
+stop Chris from sending tokens to Diana. Your front-end apps need to specify the
+addresses of all the accounts they will use.
 
-This includes the PDAs you make. Thankfully, you can calculate the address for
-PDAs in your front-end code before you write data there!
+This includes the PDAs you create. Fortunately, you can calculate the address
+for PDAs in your front-end code before writing data to them.
 
 ```typescript
-// There's nothing at this address right now, but we're going to use in our transaction
-const address = findProgramAddressSync(["seed", "another seed"], PROGRAM_ID);
+import { PublicKey } from "@solana/web3.js";
+
+// There's nothing at this address right now, but we'll use it in our transaction
+const [address, bump] = PublicKey.findProgramAddressSync(
+  [Buffer.from("seed"), Buffer.from("another seed")],
+  PROGRAM_ID,
+);
 ```
 
 ### There are multiple ways to build onchain, but we recommend Anchor
 
 You currently have two options for onchain program development:
 
-- We recommend new onchain programmers
+- We recommend new developers start with
   [start with Anchor](/content/courses/onchain-development/intro-to-anchor).
-  Anchor's defaults make it easy to create safe programs.
-- There's also a separate
-  [native onchain program development](/content/courses/native-onchain-development)
-  course.
+  Anchor's defaults make it easy to create secure programs.
+- There is also a course on
+  [native onchain program development](/content/courses/native-onchain-development).
 
-Whichever way you pick, Solana Foundation maintains
+Whichever approach you choose, the Solana Foundation provides
 [examples in both languages](https://github.com/solana-developers/program-examples),
```

Comment:
```suggestion
[examples in both Anchor and native Rust](https://github.com/solana-developers/program-examples),
```

Reply:
I have updated the link text as per a suggestion.